port/zos390/omrzfs.c

```diff
@@ -64,7 +64,7 @@ fillSyscallParamList(J9ZFSSyscallParamList *syscallParamListPtr, int32_t opcode,
  * @param[out] clientCacheSizePtr Pointer to the returned client cache area size (in bytes).
  * @return 0 on success; negative value on failure.
  */
-static int32_t
+static int32_t __attribute__((noinline))
```

Comment:
This might impact XLC builds. Can we add an `#ifdef` so that only Open XL builds are affected? Double check if `__open_xl__` is the correct flag.
```suggestion
#if defined(__open_xl__)
__attribute__((noinline))
#endif /* defined(__open_xl__) */
static int32_t
```